packages/winpanda/extra/src/winpanda/core/package/package.py

```diff
@@ -317,6 +317,7 @@ def save_manifest(self, mheading: str=None, dpath: Path=None):
         mheading = f'{mheading}: {mhd_base}' if mheading else mhd_base
 
         try:
+            cm_utl.rmdir(str(dpath), recursive=True)
```

Comment:
We don't need the `exist_ok=True` in the next line if we add this line, but that's a minor issue.

Reply:
fixed